topdown/object.go

```diff
@@ -138,6 +138,17 @@ func builtinObjectGet(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Ter
 	return iter(ast.NewTerm(value))
 }
 
+func builtinObjectKeys(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
+	object, err := builtins.ObjectOperand(operands[0].Value, 1)
+	if err != nil {
+		return err
+	}
+
+	keys := ast.NewArray(object.Keys()...)
```

Comment:
Should this return a Set instead of an Array?  The keys are going to be de-duped and ordered anyway.